### PR TITLE
Modal and text utilities

### PR DESCRIPTION
**Pull request by sushil-mod**
## Added modal and text-utils 

- modal with show /hide display 
- text -headings , center text , grey text , small text 

## screenshot 
modal
text-util

### CODE REVIEW

File: Components/text-util.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+.heading-lg{
+    font-size : 2rem;
+}
+
+.center-text{
+    text-align: center;
+}
+.grey-text{
+    color:grey;
```
Comment (jeetbhalani15) [inline Components/text-util.css]: can use color var here.

File: Components/text-util.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (Nagashree01) [inline Components/text-util.css:1]: These line are empty, you can delete them

**Review by Nagashree01 — APPROVED** [review]
Added one comment. Code is clean. Good to merge.

File: index.html [hunk 3]
```diff
@@ -624,10 +626,59 @@ <h3>To introduce Jedi UI in your project, Copy the Link given below and paste it
...
+        <p id="text-util" class="doc-header">Text Utilities</p>
+        <div class="doc-avatar flex-center">
+          <p class="doc-label">Text Utilities</p>
+          <div class="component-show  flex-col">
+              <div class="flex-center margin-btm text-gap ">
+                <div class="heading-sm">heading small </div>
+                <div class="heading-md">heading medium </div>
+                <div class="heading-lg">heading large </div>
+              </div>
+
```
Comment (jeetbhalani15) [inline index.html]: remove spaces

**Review by jeetbhalani15 — APPROVED** [review]
- Naming is Properly Done.
- No Repeated CSS.
-   Code is Clean & Readbale.

File: Components/text-util.css (hunk 2 above)
Comment (sushil-mod) [reply Components/text-util.css:1]: yes sure, thank you

File: Components/text-util.css (hunk 1 above)
Comment (sushil-mod) [reply Components/text-util.css]: yeah missed it , thanks

File: index.html (hunk 3 above)
Comment (sushil-mod) [reply index.html]: sure

**Pull request merged**